torch_geometric/nn/conv/peg_conv.py

```diff
@@ -0,0 +1,196 @@
+import math
+
+import torch
+from torch import Tensor, nn
+from torch.nn import Parameter
+from torch_sparse import SparseTensor
+
+from torch_geometric.nn.conv import MessagePassing
+from torch_geometric.nn.conv.gcn_conv import gcn_norm
+from torch_geometric.typing import Adj, OptTensor, Size, Tensor
+
+
+class PEGConv(MessagePassing):
+    r"""The PEG layer from the
+    `"Equivariant and Stable Positional Encoding
+    for More Powerful Graph Neural Networks"
+    <https://arxiv.org/abs/2203.00199>`_ paper
```

Comment:
It would be great if we can add more description here about the underlying operator :)